manifest.json

```diff
@@ -15,6 +15,10 @@
       "path": "/xero_files.yaml",
       "canPreview": true
     },
+    "finance": {
+      "path": "/xero-finance.yaml",
+      "canPreview": true
```

Comment:
@richardofortune should this be set to false like bankfeeds and identity?

Reply:
@RettBehrens @rdemarco-xero just tagged you in a Slack post about the finance API - apps won't have the requisite scopes without specifically asking for them, so we don't cater for that flow at the moment in API Explorer. I'd vote for 'false' here, but let's confirm in Slack.